### ISSUE TITLE
Crashing on Forge 1.19

### ISSUE THREAD

**Issue by ghost** [Real-Gecko/XPFromHarvest#17]
I'm using Forge 41.1.0 with 70 mods, tested basically all mods and this mod causes crashes,
Here is the crash and the latest file, also I used the 1.2.2 version of the mod:
[xp-crash.txt](https://github.com/Real-Gecko/XPFromHarvest/files/9283152/xp-crash.txt)
[xp-latest.log](https://github.com/Real-Gecko/XPFromHarvest/files/9283153/xp-latest.log)

**Comment by Real-Gecko** [comment]
If you can explain me what `InvocationTargetException` means I'll gladly fix it, it does not happen with Forge 41.0.91.

**Comment by Real-Gecko** [comment]
So, are you interested in version 1.19 or it can be 1.19.2?

**Comment by ghost** [comment]
Yeah unfortunately some of the mods I have won't work for 1.19.2, so mainly looking for 1.19 yes

**Comment by goonza93** [comment]
Same issue here...
i dont mod or develop on java, but i believe the issue is the BreakEvent class not being found (as the crash says)

Package net.minecraftforge.event.world
and now its named 
Package net.minecraftforge.event.level

in here resides the BlockEvent.BreakEvent that is failing to find when loading the mod.

i dont really know if it will still crash after that fix, but id love to get started on developing mods or fixing them so they work on my server.


Note: If you can help me setup a development enviroment id love to learn about mod development. (i work as a dev on .NET) Starting on new technology is always a long process when learning by yourself 😃 any help is appreciated.

**Comment by joshuai96** [comment]
Same with Forge `1.19.2-43.1.25`

**Comment by Real-Gecko** [comment]
Please check the latest version
